sparse/src/KokkosSparse_sptrsv_superlu.hpp

```diff
@@ -308,7 +308,8 @@ void sptrsv_symbolic(KernelHandle *kernelHandleL, KernelHandle *kernelHandleU,
 // read CrsGraph from SuperLU factor
 #ifdef KOKKOS_SPTRSV_SUPERNODE_PROFILE
   bool merge = handleL->get_merge_supernodes();
-  std::cout << " > Read SuperLU factor into KokkosSparse::CrsMatrix (invert "
+  std::cout << " > Read SuperLU factor into "
```

Comment:
What is the line length we are using to cut off these strings?

Reply:
Same as above -- I will change it back